app.R

```diff
@@ -379,7 +379,10 @@ schema_to_display_lookup <- data.frame(schema_name, display_name)
       filename_list <- names(file_namedList)
 
 
-      manifest_url <- metadata_model$getModelManifest(paste0(config$community," ", input$template_type), template_type, filenames = as.list(filename_list))
+      manifest_url <- metadata_model$getModelManifest(paste0(config$community," ", input$template_type), 
```

Comment:
@allaway I might be missing something, but it looks like the changes to the function here do not surface the [changes](https://github.com/Sage-Bionetworks/schematic/pull/456) in schematic? (The latter have been merged btw.)

Reply:
Hi @milen-sage , I removed those @sujaypatil96's recommendation given the default behavior of schematic (though I still have useAnnotations=T in my local branch). https://github.com/Sage-Bionetworks/data_curator/pull/157#discussion_r624038669